thirdparty/hnswlib/hnswlib/hnswalg.h

```diff
@@ -1250,6 +1255,9 @@ class HierarchicalNSW : public AlgorithmInterface<dist_t> {
             if (filtered_out_num >= (cur_element_count * kHnswSearchKnnBFFilterThreshold) || k >= (cur_element_count - filtered_out_num) * kHnswSearchBFTopkThreshold) {
                 return searchKnnBF(query_data, k, bitset);
             }
+#ifdef NOT_COMPILE_FOR_SWIG
+            knowhere::knowhere_hnsw_bitset_ratio.Observe((double)filtered_out_num / bitset.size());
```

Comment:
Plz put it before the BF condition check

Reply:
updated